style.css

```diff
@@ -0,0 +1,586 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,300;1,400;1,700&family=Poppins:wght@400;700&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+}
+
+.bigheader {
+  color: #d91c00;
+}
+
+.salute {
+  color: #f74d1c;
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
Since you follow mobile first approach. You don't need to use max-width value. Kindly remove this line to keep your code simple and DRY.

Reply:
@huseyinbiyik  is it mandatory?

Reply:
@Yacoubou-seidou one of the best thing about the mobile-first approach is to decrease the code amount in your CSS file. For example:
```
  .card_program h3 {
    margin-left: 20px;
    margin-right: 20px;
    color: #c74d38;
  }
```
This code repeats on your CSS file because even though you use the same values on both mobile and desktop, you need to define them again because you used max-width: 768px media query. If you wouldn't use it, you wouldn't add the same code again. Because the codes you use without max-width:768px would include each screen size but if you have a different value for desktop, you would override the general values. So you can't follow the DRY principles and that max-width:768px query at the same here. 

If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Reply:
- [ ] @Yacoubou-seidou as previously requested kindly remove this media query, it defeats the requirement of having a mobile-first approach when only code for the desktop is inside the media query